workflow/kfdrc_consensus_calling.cwl

```diff
@@ -19,10 +19,19 @@ inputs:
   output_basename: string
   annotation_vcf: {type: File, secondaryFiles: ['.tbi'], doc: "VCF of annotions to add to consensus variants, e.g. gnomAD allele frequency"}
   vep_cache: {type: File, doc: "tar gzipped cache from ensembl/local converted cache"}
+  annot_columns: {type: string?, default: 'INFO/AF', doc: "column from annotation_vcf to add to consensus VCF; defaults to 'INFO/AF'"}
+  filter_names: {type: 'string[]?', default: [ "NORM_DP_LOW", "GNOMAD_AF_HIGH" ], doc: "Names of filters to be added to consensus VCF;\
+     \ defaults to [ 'NORM_DP_LOW', 'GNOMAD_AF_HIGH' ]"}
+  bcftools_public_filter: {type: string?, doc: 'Will hard filter final result to create a public version, e.g. FILTER="PASS"|INFO/Hotspot=1', default: 'FILTER="PASS"|INFO/Hotspot=1'}
```

Comment:
it seems you are missing `filter_expression`, which is needed to properly add soft filters to the vcf. Otherwise, nothing happens during the public filter portion. Could rename this variable, since we have the `bcftools_public_filter` from `filter_names` to something like `soft_filter_names` and `soft_filter_expression`. I might want to do that myself at some point to add clarity to the annotation workflow...

Reply:
Are you referring to the argument for which the defaults are 
[ "vc.getGenotype(" + inputs.input_normal_name + ").getDP() <= 7", "AF > 0.001" ]
?

These are the ones for which I was having trouble getting it to work because of the normal_name dependency. On this branch, those filters are still set in the variant_filter step, but there is no ability to change the defaults. I will fix this on my other branch as soon as I figure out how if that's ok.

Reply:
Ah, yes, I missed that. Which points me to the real bug, I see this error in the log for that filter step:
`2021-04-15T19:50:46.598474932Z sh: 1: input_normal_name: not found`
resulting from this arg (copied from the cmd.log): ` --filter-expression "vc.getGenotype($(input_normal_name)).getDP() <= 7"` not being resolved properly